yannis_tether_control_node: # $USER + tether_control_node, @todo: change
  ros__parameters:
    log_mode: true
    debug_mode: true
    gravComp: 9.81 # z acceleration to compensate for hover, to move to a define
    tethered: false
    uav_type: "MC" # possibilities: MC, VTOL, VTOL-TAILSITTER
    control:
      control_mode: "CUSTOM" #TETHER_FORCE_REACTIONS, ATTITUDE, THRUST_IDENTIF
      hoverThrust: 0.3733 # empirically determined, thrust to hover in sim 0.3733
      thrust_force_constant: 7.17420305 # empirically: k = F/thrust = 1.638/0.3733 = 7.17420305
      attThrustKp: 0.5 # empirically determined, Kp for attitude control
      attThrustKd: 0.05 # empirically determined, Kd for attitude control
      attR: 0.0
      attP: 0.0
      attY: 0.0
    model:
      disturb_mode: "TET_GRAV_FIL_ANG" # possibilities: STRONG_SIDE, CIRCULAR, TET_GRAV_FIL_ANG
      tether_init_length: 10.0 # [m]
      tether_total_length: 100.0 # [m]
      tether_diameter: 0.01 # [m]
      tether_density: 970.0 #source: Sofia [kg/mÂ³]
      winch_diameter: 0.1 # [m]